Samples/Server/NativeServer.Tests/NativeServersTests.cpp

```diff
@@ -442,3 +442,95 @@ TEST(BufferCapturerTests, CaptureFrameStereoUsingDirectXBufferCapturer)
 			capturer->staging_frame_buffer_.Get(), 0);
 	}
 }
+
+// --------------------------------------------------------------
+// Decoder tests
+// --------------------------------------------------------------
+TEST(RtpHeaderFramePredictionTest, CanPredictCorrectly) {
```

Comment:
CanPredictCorrectly might be a bit misleading, let's rename to SendReceiveFramePredictionTimestamps